src/handlers/parseRawDependencyDataHandler.ts

```diff
@@ -48,3 +50,20 @@ function parseTreeNodes(treecontent: string, separator: string, indent: string,
     }
     return treeNodes;
 }
+
+function setOldestAncestor(treeNodes: Dependency[]): void {
```

Comment:
Not necessarily to update `node.root` separately after parsing. You can do it during parsing.